.github/workflows/noora.yml

```diff
@@ -1,13 +1,19 @@
-name: Noora
+name: Noora CLI
 
 on:
   push:
     branches:
       - main
-  pull_request: {}
+    paths:
+      - 'packages/cli/**'
```

Comment:
I would keep the `cli` at the top level, so we end up with:
```
/cli
/web
```

Reply:
I still somewhat strongly to remove the `packages` layer we don't actually leverage. It feels adding extra indirection for not a real benefit 🤔

Reply:
Yeah same here. It makes the sides of Noora more prominent when looking at the root of directories on GitHub's UI.